src/components/Explore/utils.ts

```diff
@@ -361,6 +366,22 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.VACCINATIONS_BIVALENT_FALL_2022]: {
+    title: '% Vaccinated (bivalent booster)',
+    name: 'Percent vaccinated (bivalent booster)',
+    chartId: 'vaccinations_bivalent_dose_fall_2022',
```

Comment:
And here you're not matching what you did above in getMetricByChartId and getDatasetIdByMetric (with it being `vaccinations` now I think)...

Reply:
I'm not really sure what these chart Ids are for tbh... 

I think we use `datasetId` (below) to lookup with `getDatasetIdByMetric`, but I'm not sure what the `chartId` corresponds to as they don't appear to be used anywhere